src/conspiracies/docprocessing/relationextraction/multi2oie/multi2oie_utils.py

```diff
@@ -111,3 +112,12 @@ def match_extraction_spans_to_wp(
 
         matched_extractions += new_spans
     return matched_extractions
+
+
+_tokenizer_cache = {}
+
+
+def get_cached_tokenizer(model_name):
```

Comment:
looks good, but you could probably do with simply recording the `BertTokenizer.from_pretrained` using the cache decorator.

```
from functools import cache # 
get_cached_tokenizer = cache(BertTokenizer.from_pretrained)
```

Reply:
That was the kind of thing I was looking for. I'll try it. Thanks!